### DISCUSSION TITLE
IdeaVim License Change: Associating GitHub Accounts to the Commit Authors

### DISCUSSION THREAD

**Discussion by AlexPl292** [JetBrains/ideavim#541] [License]
The IdeaVim project is going to be relicensed from GPL-2.0 or later to the MIT license. To do that properly, we need to obtain consent from the contributors. However, for some of the contributors, we don’t have information about an associated GitHub account.

To learn more about the license change, please visit [“IdeaVim License Change: Discussion”](https://github.com/JetBrains/ideavim/discussions/543)
Consent to license change is collected in [“IdeaVim License Change: Obtaining Consent”](https://github.com/JetBrains/ideavim/discussions/542)

We kindly ask everyone mentioned in this table to leave a comment if you’re the author of the commits to the IdeaVim repository. If you are, please respond with:

> I am the author of all IdeaVim commits from [YOUR_EMAIL].

|  Contributor Email                 | Possible GitHub Account |
|-------------------------------|-------------------------|
| smartbomb@server.fake         | @smartbomb              |
| agrsbm@gmail.com              | @agriesbaum             |
| jflorian@doubledog.org        | @jflorian               |
| hild@b4mad.net                | @durandom               |
| manwe64@gmail.com             | @baldrs                 |

Once you’ve done this, we’ll remove you from the table and add you to the [“IdeaVim License Change: Obtaining Consent”](https://github.com/JetBrains/ideavim/discussions/542) discussion. To avoid useless delays, you may respond in that discussion without waiting to be mentioned explicitly.

## Unsubscribe from this discussion

Since this discussion is going to be used only for identifying authors, you may want to unsubscribe from it after you respond.

![image](https://user-images.githubusercontent.com/4203721/187895393-62fbe21c-1839-4919-b744-3c2f8f4ec1bf.png)

**Comment by raimon49** [comment]
@AlexPl292 Thanks for the notification.

I am the author of all IdeaVim commits from raimon49@hotmail.com, and I agree with the license change.

**Reply by AlexPl292** [reply]
Thank you for the response! I'd really appreciate if you could response to the https://github.com/JetBrains/ideavim/discussions/542 with the form attached.

**Reply by raimon49** [reply]
I have responded.

**Comment by zyv** [comment]
I am the author of all IdeaVim commits from yury@shurup.com .

**Comment by strogiyotec** [comment]
I am the author of all IdeaVim commits from almas337519@gmail.com

**Reply by AlexPl292** [reply]
Could you please also leave a comment in this discussion? Thank you. https://github.com/JetBrains/ideavim/discussions/542

**Comment by dezgeg** [comment]
I am the author of all IdeaVim commits from tuomas.tynkkynen@iki.fi

**Comment by madgnome** [comment]
I am the author of all IdeaVim commits from madgnome@gmail.com

**Comment by homburg** [comment]
I am the author of all IdeaVim commits from thomas@homburg.dk .

**Comment by dathanb** [comment]
I am the author of all IdeaVim commits from dbennett@palantir.com.

**Comment by abrookins** [comment]
I am the author of all IdeaVim commits from a.m.brookins@gmail.com.

**Comment by rasendubi** [comment]
I am the author of all IdeaVim commits from [cb-ashmalko@jawbone.com](mailto:cb-ashmalko@jawbone.com)

**Comment by jonatantierno** [comment]
I am the author of all IdeaVim commits from jonatantierno@gmail.com, jta.javato@gmail.com and jtierno@tuenti.com.

**Comment by jdpopkin** [comment]
I am the author of all IdeaVim commits from jackson@donorschoose.org.

**Comment by cypok** [comment]
I am the author of all IdeaVim commits from [vparfinenko@excelsior-usa.com](mailto:vparfinenko@excelsior-usa.com)

**Comment by nickgieschen** [comment]
I am the author of all IdeaVim commits from nickgieschen@gmail.com

**Comment by changwang** [comment]
I am the author of all IdeaVim commits from changwang83@gmail.com

**Reply by AlexPl292** [reply]
Thank you for your reply. Could you please also leave a comment in this discussion? https://github.com/JetBrains/ideavim/discussions/542

**Comment by awkay** [comment]
I am the author of all IdeaVim commits from tony.kay@gmail.com

**Comment by jamescmartinez** [comment]
I am the author of all IdeaVim commits from jamescmartinez@gmail.com

**Reply by AlexPl292** [reply]
Thank you for your reply. Could you please also leave a comment in this discussion? https://github.com/JetBrains/ideavim/discussions/542

**Comment by luba239** [comment]
I am the author of all IdeaVim commits from lubashka.994@mail.ru
